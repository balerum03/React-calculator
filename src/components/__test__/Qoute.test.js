import React from 'react';
import renderer from 'react-test-renderer';
import Qoute from '../Quote';

test('Component renders correctly', () => {
  const component = renderer.create(
    <Qoute />,
  ).toJSON();
  expect(component).toMatchSnapshot();
});
