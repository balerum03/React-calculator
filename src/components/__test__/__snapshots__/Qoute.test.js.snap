// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders correctly 1`] = `
<div>
  <h2
    className="quote"
  >
    Mathematics is not about numbers, equations, computations, or algorithms:
  </h2>
  <h2>
    it is about understanding. â€“William Paul Thurston
  </h2>
</div>
`;
